pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -95,7 +95,7 @@ let
   browserPatches = [
     ./env_var_for_system_dir.patch
   ]
-  ++ lib.optional (lib.versionAtLeast ffversion "63" && lib.versionOlder ffversion "69")
+  ++ lib.optional (lib.versionAtLeast ffversion "63" && lib.versionOlder ffversion "68.2.0")
```

Comment:
This should be "68.3.0". This is my mistake.

Reply:
Fixed.